cli/info/pretty.go

```diff
@@ -0,0 +1,123 @@
+package info
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+
+	"github.com/nknorg/nkn/pb"
+)
+
+type PrettyPrinter []byte
+
+func TxnUnmarshal(m map[string]interface{}) (interface{}, error) {
+	typ, ok := m["txType"]
+	if !ok {
+		return nil, fmt.Errorf("No such key [txType]")
+	}
+
+	pbHexStr, ok := m["payloadData"]
+	if !ok {
+		return m, nil
+	}
+	buf, err := hex.DecodeString(pbHexStr.(string))
+
+	switch typ {
+	case "SIG_CHAIN_TXN_TYPE":
```

Comment:
It's better to use PayloadType_name[SIG_CHAIN_TXN_TYPE] to dynamically get string instead of hard coding string here. Same for other cases below

Reply:
Done.